smtchecker.ts

```diff
@@ -3,7 +3,7 @@
 // The function runs an SMT solver on each query and adjusts the input for
 // another run.
 // Returns null if no solving is requested.
-function handleSMTQueries (inputJSON, outputJSON, solverFunction, solver) {
+function handleSMTQueries (inputJSON: any, outputJSON: any, solverFunction?: any, solver?: any) {
```

Comment:
@leonardoalt are all functions required or is `solverFunction`/`solver` still optional?

Reply:
I added the arguments as `null` where they're not needed, so I guess they could be mandatory, as long as `null` is acceptable